src/FormDecorator/IcingaFormDecorator.php

```diff
@@ -71,6 +72,12 @@ protected function assembleLabel()
         $label = parent::assembleLabel();
         if ($label !== null && ! $this->formElement instanceof FieldsetElement) {
             $label->addWrapper(new HtmlElement('div', Attributes::create(['class' => 'control-label-group'])));
+        } else {
+            $label = new HtmlElement(
+                'div',
+                Attributes::create(['class' => 'control-label-group']),
+                HtmlString::create('&nbsp')
+            );
```

Comment:
This must not happen for FieldsetElements! They either have a legend or nothing.

Then you can add the monthly and annually fields by `addElement` in `ScheduleElement`.